command_test.go

```diff
@@ -15,3 +19,70 @@ func TestCommand(t *testing.T) {
 		t.Errorf("Expected %d commands, got %d", numberOfCommands, len(CommandTypes))
 	}
 }
+
+func TestExecuteSetTheme(t *testing.T) {
+	browser := rod.New().MustConnect()
```

Comment:
These tests seem a bit heavy for `go test` (i.e. don't want to need a browser installed to run tests), we already get integration testing by generating the GIFs. Can we just run `ApplyTheme` to check the go struct is updated?